docs/pin.rst

```diff
@@ -23,57 +24,65 @@ depending upon the digital reading on pin 0::
-|  10 | Analog  | Column 3 |
-+-----+---------+----------+
-|  11 | Digital | Button B |
-+-----+---------+----------+
-|  12 | Digital |          |
-+-----+---------+----------+
-|  13 | Digital | SPI SCK  |
-+-----+---------+----------+
-|  14 | Digital | SPI MISO |
-+-----+---------+----------+
-|  15 | Digital | SPI MOSI |
-+-----+---------+----------+
-|  16 | Digital |          |
-+-----+---------+----------+
-+-----+---------+----------+
-|  19 | Digital | I2C SCL  |
-+-----+---------+----------+
-|  20 | Digital | I2C SDA  |
-+-----+---------+----------+
++-----+---------+----------+-----------------+
+| Pin | Type    | Function | Function **V2** |
++=====+=========+==========+=================+
+|  0  | Touch   | Pad 0    | Pad 0           |
++-----+---------+----------+-----------------+
+|  1  | Touch   | Pad 1    | Pad 1           |
++-----+---------+----------+-----------------+
+|  2  | Touch   | Pad 2    | Pad 2           |
++-----+---------+----------+-----------------+
+|  3  | Analog  | Column 1 | Column 3        |
++-----+---------+----------+-----------------+
+|  4  | Analog  | Column 2 | Column 1        |
++-----+---------+----------+-----------------+
+|  5  | Digital | Button A | Button A        |
++-----+---------+----------+-----------------+
+|  6  | Digital | Column 9 | Column 4        |
++-----+---------+----------+-----------------+
+|  7  | Digital | Column 8 | Column 2        |
++-----+---------+----------+-----------------+
+|  8  | Digital |          |                 |
++-----+---------+----------+-----------------+
+|  9  | Digital | Column 7 |                 |
++-----+---------+----------+-----------------+
+|  10 | Analog  | Column 3 | Column 5        |
++-----+---------+----------+-----------------+
+|  11 | Digital | Button B | Button B        |
++-----+---------+----------+-----------------+
+|  12 | Digital |          |                 |
++-----+---------+----------+-----------------+
+|  13 | Digital | SPI SCK  | SPI SCK         |
++-----+---------+----------+-----------------+
+|  14 | Digital | SPI MISO | SPI MISO        |
++-----+---------+----------+-----------------+
+|  15 | Digital | SPI MOSI | SPI MOSI        |
++-----+---------+----------+-----------------+
+|  16 | Digital |          |                 |
++-----+---------+----------+-----------------+
++-----+---------+----------+-----------------+
+|  19 | Digital | I2C SCL  | I2C SCL         |
++-----+---------+----------+-----------------+
+|  20 | Digital | I2C SDA  | I2C SDA         |
++-----+---------+----------+-----------------+
 
 The above table summarizes the pins available, their types (see below) and what
 they are internally connected to.
 
+The latest micro:bit device **V2** has two additional pins that you can access
+in MicroPython, but that are not available via the edge connector:
+
+- ``pin_logo`` - A touch sensitive logo pin on the front of the micro:bit,
+which by default is set to capacitive touch mode.
```

Comment:
This is still not rendered correctly (too mush space between lines):
![image](https://user-images.githubusercontent.com/29712657/100361257-d9a49d80-2ff1-11eb-9172-d4bcf1ea50e1.png)

Reply:
Not sure what to do here. I've changed `-` to `*` to see if it has an effect on the ordered list.

Reply:
Indentation should fix this

Reply:
Fixed!
![image](https://user-images.githubusercontent.com/29712657/100443300-5b9fcf80-30a1-11eb-9f2b-8a6ca5a4e253.png)